tidb-lightning/tidb-lightning-table-filter.md

```diff
@@ -1,129 +0,0 @@
----
```

Comment:
Is it deleted because this file is duplicated by the [table-filter.md](https://github.com/pingcap/docs/blob/release-3.1/table-filter.md) file?

Reply:
Yes. This file should have been deleted in this PR (https://github.com/pingcap/docs/pull/3139), but the cherry-pick bot just missed it.

Reply:
Got it.